mod-consortia/src/main/resources/karate-config.js

```diff
@@ -0,0 +1,166 @@
+    },
+
+    randomMillis: function() {
+      return java.lang.System.currentTimeMillis() + '';
+    },
+
+    random_string: function() {
+      var text = "";
+      var possible = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
+      for (var i = 0; i < 5; i++)
+        text += possible.charAt(Math.floor(Math.random() * possible.length));
+      return text;
+    },
+    getCurrentYear: function() {
+      var SimpleDateFormat = Java.type('java.text.SimpleDateFormat');
+      var sdf = new SimpleDateFormat('yyyy');
+      var date = new java.util.Date();
+      return sdf.format(date);
+    },
+    getCurrentDate: function() {
+      var SimpleDateFormat = Java.type('java.text.SimpleDateFormat');
+      var sdf = new SimpleDateFormat('yyyy-MM-dd');
+      var date = new java.util.Date();
+      return sdf.format(date);
+    },
+
+    getYesterday: function() {
+      var LocalDate = Java.type('java.time.LocalDate');
+      var localDate = LocalDate.now().minusDays(1);
+      var formatter = java.time.format.DateTimeFormatter.ofPattern("yyyy-MM-dd");
+      var formattedString = localDate.format(formatter);
+      return localDate.format(formatter);
+    },
+
+    pause: function(millis) {
+      var Thread = Java.type('java.lang.Thread');
+      Thread.sleep(millis);
+    }
+
+  };
+
+  // Create 100 functions for uuid generation
+  var rand = function(i) {
+    karate.set("uuid"+i, function() {
+      return java.util.UUID.randomUUID() + '';
+    });
+  }
+  karate.repeat(100, rand);
+
+  if (env == 'snapshot-2') {
+    config.baseUrl = 'https://folio-snapshot-2-okapi.dev.folio.org:443';
+    config.edgeUrl = 'https://folio-snapshot-2.dev.folio.org:8000';
+    config.ftpUrl = 'ftp://ftp.ci.folio.org';
+    config.ftpPort = 21;
+    config.ftpUser = 'folio';
+    config.ftpPassword = 'Ffx29%pu';
+    config.admin = {
+      tenant: 'supertenant',
+      name: 'testing_admin',
+      password: 'admin'
+    }
+  } else if (env == 'snapshot') {
+    config.baseUrl = 'https://folio-snapshot-okapi.dev.folio.org:443';
+    config.edgeUrl = 'https://folio-snapshot.dev.folio.org:8000';
+    config.ftpUrl = 'ftp://ftp.ci.folio.org';
+    config.ftpPort = 21;
+    config.ftpUser = 'folio';
+    config.ftpPassword = 'Ffx29%pu';
+    config.admin = {
+      tenant: 'supertenant',
+      name: 'testing_admin',
+      password: 'admin'
+    }
+  } else if (env == 'rancher') {
+    config.baseUrl = 'https://folio-dev-thunderjet-okapi.ci.folio.org';
+    config.edgeUrl = 'https://folio-snapshot.dev.folio.org:8000';
+    config.ftpUrl = 'ftp://ftp.ci.folio.org';
+    config.ftpPort = 21;
+    config.ftpUser = 'folio';
+    config.ftpPassword = 'Ffx29%pu';
+    config.prototypeTenant= 'diku'
+    config.admin = {
+      tenant: 'diku',
+      name: 'diku_admin',
+      password: 'admin'
+    }
+  } else if (env == 'thunderjet-consortia-rancher') {
+    config.baseUrl = 'https://folio-dev-thunderjet-okapi.ci.folio.org:443';
```

Comment:
This is the same as line 112. Also for admin credentials is not up to date, maybe it is better to get them from env as they maybe changed in future (maybe better not to expose them).

Reply:
changed to variable